### DISCUSSION TITLE
Is it possible to have Chromium bundled/used for a Linux build ?

### DISCUSSION THREAD

**Discussion by glinford** [tauri-apps/tauri#5796] [Q&A] [answered]
Hi,

Is it possible to have Chromium bundled/used for a Linux build ?

Looking at the document it appears it's not supported out of the box
https://tauri.app/v1/guides/building/linux
https://tauri.app/v1/references/webview-versions/

But since sideCars are a thing, is there a way to force Chromium rendering engine rather than WebKit on Linux ? (even at the risk of increasing bundle size)

Have this been done with Tauri before ?

A similar questions was asked before but lacked a bit details https://github.com/tauri-apps/tauri/discussions/4591 and I have no access to Discord.

Kind regards and Thanks!

**Comment by FabianLars** [comment, marked as answer]
No.

sidecars also don't help at all here. It would basically allow us to bundle a "normal" browser, but we can't use browsers as a tauri backend because we don't have any control over them.

For what it's worth, we are somewhat actively researching our own webview which _could_ be based on chromium too.

**Reply by Sytten** [reply]
This is something we would be very interested to use for our product. This would be a big plus for companies looking to switch from electron. Specially since the gtk-webview performances are so bad.

**Reply by tbontb-iaq** [reply]
So can there be a public chromium-based webview on Linux and Mac platforms instead of WebKit, so that developers do not need to package chrome in the application and do not have to worry about browser compatibility issues?

**Reply by FabianLars** [reply]
This would be the ideal solution but it's far from trivial.
